test/integration/smart_answer_flows/uk_benefits_abroad_test.rb

```diff
@@ -476,6 +476,15 @@ class UKBenefitsAbroadTest < ActiveSupport::TestCase
           end
         end
       end
+
+      context "answer Kosovo" do # SS country
```

Comment:
Similar point about fixing up to previous commit.

Also looks like it applies to the next four commits or so.